java/client/src/org/openqa/selenium/json/JsonOutput.java

```diff
@@ -65,38 +65,6 @@
     GSON_ELEMENT = gsonElement;
   }
 
-  // https://www.json.org has some helpful comments on characters to escape
-  // See also https://tools.ietf.org/html/rfc8259#section-7 and
-  // https://github.com/google/gson/issues/341 so we escape those as well.
-  // It's legal to escape any character, so to be nice to HTML parsers,
-  // we'll also escape "<" and "&"
-  private static final Map<Integer, String> ESCAPES;
```

Comment:
Prefer keeping this static block and setting a boolean for escaping if we're heading down this route.